nixos/doc/manual/release-notes/rl-2205.section.md

```diff
@@ -378,6 +378,8 @@ In addition to numerous new and upgraded packages, this release has the followin
 
 - `systemd-nspawn@.service` settings have been reverted to the default systemd behaviour. User namespaces are now activated by default. If you want to keep running nspawn containers without user namespaces you need to set `systemd.nspawn.<name>.execConfig.PrivateUsers = false`
 
+- `systemd-shutdown` is now properly linked on shutdown to unmount all filesystems and device mapper devices cleanly. This can be disabled using `boot.systemd.shutdown.enable`.
```

Comment:
This only applies if you opt in to use systemd in initrd, right? Maybe the release notes should reflect that.

Reply:
No, that also works with the old initrd. There is nothing that relates to the new one